api/routes/family.py

```diff
@@ -190,11 +190,21 @@ async def import_families(
 
     family_layer = FamilyLayer(connection)
     reader = csv.reader(codecs.iterdecode(file.file, 'utf-8-sig'), delimiter=delimiter)
+    rows = [r for r in reader if not r[0].startswith('#')]
+
+    if len(rows) == 0:
+        if has_header:
+            return {
+                'success': False,
+                'message': 'A header was expected but file is empty.',
+            }, 400
+        return {}, 204
```

Comment:
Also, not sure if 204 is ok to use here, MDN docs say:

> Note that the response must not include any content or the [Content-Length](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Content-Length) header (browsers may reject responses that include content).

https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/204

So sounds like this could cause errors if this was used in a browser?

Reply:
I think a 200 would probably be fine